event.c

```diff
@@ -627,9 +629,54 @@ static void event_ffmpeg_newfile(struct context *cnt,
      *  motion movies get the same name as normal movies plus an appended 'm'
      *  PATH_MAX - 4 to allow for .mpg to be appended without overflow
      */
-    snprintf(cnt->motionfilename, PATH_MAX - 4, "%s/%sm", cnt->conf.filepath, stamp);
-    snprintf(cnt->newfilename, PATH_MAX - 4, "%s/%s", cnt->conf.filepath, stamp);
-
+     /* Set up a testing / evaluation codec which will use a different
+      * codec for the events.
+     */
```

Comment:
I'd possibly expand the comment a little to explain that setting the codec to "test" means that it cycles through all the codecs, switching at the end of each event.